diagnostic.md

```diff
@@ -7,27 +7,31 @@ Write your answers inside this file, where it's indicated by the comments.
     repository in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    Once you forked it first you want to navigate your way to the correct folder and use the command `git clone <ssh link copied from github.>`
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    You would want to first cd into the repository and from there create a new branch that is seperate from the master branch.
     ```
 
 3.  What command do we use to create a new branch? Name this branch `response`
     in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    there are two ways of doing it:
+    1. git branch <name of branch>
+        -but this does not checkout the branch
+    2. git checkout -b <name of branch>
+        - this checks out the branch and creates a new at the same time
     ```
 
 4.  We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    After staging and committing the changes with messages, you can use `git log`to see the history of changes.
```

Comment:
No, `git log` will show you the previous commits that have been made; it will *not* show you outstanding changes in the working directory.